nexus/db-model/src/schema.rs

```diff
@@ -1512,6 +1512,19 @@ table! {
         cockroachdb_fingerprint -> Text,
 
         cockroachdb_setting_preserve_downgrade -> Nullable<Text>,
+
+    }
+}
+
+table! {
+    bp_clickhouse_cluster_config (blueprint_id) {
```

Comment:
We really only want one of these for most blueprints, unless we are adding or removing clickhouse related zones. We should probably make the primary key `generation`, as the `blueprint_id` indicates what blueprint it was generated in. We'll want to just lookup the latest one at all times.